SYCL/BFloat16/bfloat16_example_jit.cpp

```diff
@@ -0,0 +1,15 @@
+///
+/// Check if bfloat16 example works using fallback libraries in JIT mode
+///
+
+// REQUIRES: gpu
```

Comment:
it's slightly strange to require a certain device for JIT compilation

Reply:
The intent is to test JIT'ing for GPU.